platform/default/mbgl/storage/offline_database.cpp

```diff
@@ -126,6 +127,14 @@ void OfflineDatabase::migrateToVersion5() {
     db->exec("PRAGMA user_version = 5");
 }
 
+void OfflineDatabase::migrateToVersion6() {
```

Comment:
Good point. Before I opted to use `ALTER TABLE`, I looked up the [SQLite docs and they say this](https://sqlite.org/lang_altertable.html):

> The execution time of the `ALTER TABLE` command is independent of the amount of data in the table. The `ALTER TABLE` command runs as quickly on a table with 10 million rows as it does on a table with 1 row.

I also conducted a test with a database that was ~300 MB in size and contained 6000 tiles. The query execution time the SQLite3 command line showed was < 1 millisecond.